pkgs/development/python-modules/markdown2/default.nix

```diff
@@ -29,6 +33,16 @@ buildPythonPackage rec {
     runHook postCheck
   '';
 
+  passthru.optional-dependencies = {
+    code_syntax_highlighting = [
+      pygments
+    ];
+    wavedrom = [
+      wavedrom
+    ];
+    all = [ pygments wavedrom ];
```

Comment:
```suggestion
    all = lib.flatten (lib.attrValues (lib.filterAttrs (n: v: n != "all") passthru.optional-dependencies))
```

Reply:
I'm not a huge fan of this, but I don't really want to spend a lot of time on this PR, so I just went with it.